### PR TITLE
🇰🇷 update for korean: Add word for grammatical readability 

### PR DESCRIPTION
**Pull request by PreviousW**
add word "기능을" behind the phrase: '%s'

### CODE REVIEW

**Review by arm32x — APPROVED** [review]
Looks good to me.

**Comment by PreviousW** [comment]
> Looks good to me.

> thanks! I hope you merge this. :)

**Comment by dytroc** [comment]
As the word "기능" could be translated in English as "function", it could be misleading for those who are not used to Korean that this translation is valid.

However, the word "기능" has a meaning of "function" as used in "The main function of DNA is..."

On the other hand, the word "function" used here refers to the functions of data packs, which derive from the concept of functions from programming. Therefore, I suggest using the word "함수" instead of "기능" like how the official Crowdin of Minecraft translates functions in data packs.

File: src/main/resources/assets/commandblockide/lang/ko_kr.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
...
-  "commandBlockIDE.saveFunction.failed.ioException": "'%s' 저장하는 도중 오류 발생: I/O 문제가 발생했습니다.",
-  "commandBlockIDE.saveFunction.failed.zipNotSupported": "'%s' 저장하는 도중 오류 발생: '.zip' 형태의 데이터팩은 아직 지원하지 않습니다.",
-  "commandBlockIDE.saveFunction.failed.packClassNotSupported": "'%s' 저장하는 도중 오류 발생: '%s' 형태의 데이터팩은 지원하지 않습니다.",
-  "commandBlockIDE.saveFunction.failed.invalidPath": "'%s' 저장하는 도중 오류 발생: 잘못된 파일 경로 '%s': %s"
+  "commandBlockIDE.saveFunction.failed.missingNamespace": "'%s' 기능을 저장하는 도중 오류 발생: '%s'라는 네임스페이스가 없습니다.",
+  "commandBlockIDE.saveFunction.failed.noResourcePack": "'%s' 기능을 저장하는 도중 오류 발생: 데이터팩을 찾을 수가 없습니다.",
+  "commandBlockIDE.saveFunction.failed.ioException": "'%s' 기능을 저장하는 도중 오류 발생: I/O 문제가 발생했습니다.",
+  "commandBlockIDE.saveFunction.failed.zipNotSupported": "'%s' 기능을 저장하는 도중 오류 발생: '.zip' 형태의 데이터팩은 아직 지원하지 않습니다.",
+  "commandBlockIDE.saveFunction.failed.packClassNotSupported": "'%s' 기능을 저장하는 도중 오류 발생: '%s' 형태의 데이터팩은 지원하지 않습니다.",
+  "commandBlockIDE.saveFunction.failed.invalidPath": "'%s' 기능을 저장하는 도중 오류 발생: 잘못된 파일 경로 '%s': %s"
```
Comment (dytroc) [inline src/main/resources/assets/commandblockide/lang/ko_kr.json]: It would be better to translate it as "함수 '%s'을(를) 저장하는 도중 오류 발생:".

Comment (PreviousW) [reply src/main/resources/assets/commandblockide/lang/ko_kr.json]: On second thought, it goes more smoothly.

**Comment by PreviousW** [comment]
hmm.?

**Comment by arm32x** [comment]
Sorry, I approved this intending to merge it later and then forgot about it. It's merged now.

**Pull request merged**